src/hooks/useCommands.tsx

```diff
@@ -22,61 +21,6 @@ interface IParam extends ObjectWithStringKeys {
   version: string
 }
 
-export const useCommands = () => {
```

Comment:
This got moved further down the file. Otherwise it's unchanged. Due to an abandoned plan but mistakenly didn't put it back where it belonged :(